trust-sdk/trust-sdk-specs.md

```diff
@@ -9,14 +9,50 @@ All commands follow the URL format below:
 Where: 
 * `[scheme]` is the wallet's deeplink scheme. Default to `trust`
 * `[command]` is the command to be executed
-* `[params]` is the command parameters encoded as url query items.
+* `[params]` is the command parameters encoded as url query items
 * `[id]` is the unique command id. The value is later used to resolve the callback for the command. Can be any incrementing integer.
 * `[app]` callback URL scheme. 
-* `[callback]`callback path. Default to `sgn_sign_result`
+* `[callback]`callback path. Default is `sgn_sign_result`
 
-### Sign
+Trust will open following URL format if command is succeeded:
 
-Sign command is used to sign and broadcast transactions leveraging [wallet core](/wallet-core/wallet-core.md) protobuf models (encoded in base64).
+`[app]://[callback]?[id]&[params]`
+
+Where: 
+* `[params]` is the response parameters, see `Parameters` section below for each command
+
+Error response format:
+
+`[app]://[callback]?[id]&error=[error type]&message=[error message]`
+
+### Get Accounts
+
+#### Command
+
+* Command: `sdk_get_accounts`
+* Parameters:
+  * `coins`: BIP44 code array encoded as URL params. Check how it's encoded in [Dictionary Encoding](#dictionary-encoding).
+
+##### Example
+
+```shell
+trust://sdk_get_accounts?coins.0=60&coins.1=714&app=sampleapp&callback=sdk_sign_result&id=2
```

Comment:
Why we should separate app and callback? callback will contains app schema or app link.

Reply:
explained here: https://github.com/trustwallet/developer/pull/150#discussion_r519706788